java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -3088,7 +3114,7 @@ public long getColumnViewAddress() {
 
   @Override
   public ColumnViewAccess getChildColumnViewAccess(int childIndex) {
-    if (!type.isNestedType()) {
+    if (!type.typeId.isNestedType()) {
```

Comment:
I would parrot @jlowe 's comment here to add the `isNestedType` to DataType to reduce code changes

Reply:
Done